benchmarks/htap/htap_loader.py

```diff
@@ -0,0 +1,306 @@
+from datetime import datetime
+from io import StringIO, SEEK_SET
+
+from psycopg2.extras import execute_values
+
+from s64da_benchmark_toolkit.dbconn import DBConn
+
+from benchmarks.htap.lib.helpers import (
+        Random, TPCCText, TPCHText, NATIONS, REGIONS, TimestampGenerator
+)
+
+MAXITEMS = 100000
+DIST_PER_WARE = 10
+CUST_PER_DIST = 3000  # Standard says: 30k customers total (for all districts)
```

Comment:
In that case, why not make it: `CUST_PER_DIST = 30000 / DIST_PER_WARE` ?

Reply:
done